commons/connector/fitbit/fitbit_food_log.avsc

```diff
@@ -0,0 +1,108 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitFoodLog",
+  "doc": "A summary of the user's food log entry for a given day.",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)."
+    },
+    {
+      "name": "isFavorite",
+      "type": "boolean",
+      "doc": "Whether the food is the favorite one."
+    },
+    {
+      "name": "logId",
+      "type": "long",
+      "doc": "The id of the logged food."
+    },
+    {
+      "name": "accessLevel",
+      "type": "string",
+      "doc": "The access level of the food."
```

Comment:
Not sure what this means...

Reply:
## Additional Information

### Access Levels
There are three access level types for food log entries that an authorized user can view via API requests. Each food is annotated with an accessLevel field with one of the following values:

- PUBLIC - Foods that are in Fitbit's public food database and are visible to any Fitbit users. Only Fitbit populates this database to avoid spam and duplicate entries.
- PRIVATE - Foods created by a user either on the website or via the [Create Food](https://dev.fitbit.com/build/reference/web-api/nutrition/create-food/) endpoint.
- SHARED - A food created by a user whose foods privacy is set to Friends or Anyone. These can be logged either on the website or via the [Create Food](https://dev.fitbit.com/build/reference/web-api/nutrition/create-food/) endpoint. These foods can be discovered using the [Search Foods](https://dev.fitbit.com/build/reference/web-api/nutrition/search-foods/) endpoint.

### Considerations
The units field contains references to respective Food Units.
The accessLevel field can be PUBLIC, PRIVATE, or SHARED.
The mealTypeId field is always 7 (Anytime). Meals are not associated with particular times.